controllers/scripts/player.gd

```diff
@@ -1,39 +1,43 @@
 class_name Player
 extends CharacterBody3D
 
+enum STATE {ACTIVE, IN_MENU}
+
+const TILT_LOWER_LIMIT := deg_to_rad(-90.0)
```

Comment:
A lot of these lines are just re-arranged, per https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_styleguide.html